### ISSUE TITLE
[MEV Slicer] - consistent naming

### ISSUE THREAD

**Issue by MareenG** [gnosis/cowswap#1780]
**Description**

In the Transaction submitted modal the mev slicer is named cow slicer. 
<img width="719" alt="Bildschirmfoto 2021-11-04 um 13 37 42" src="https://user-images.githubusercontent.com/34510341/140314998-9f6a5b22-ecf0-42bb-a171-817d868e1745.png">

<img width="329" alt="Bildschirmfoto 2021-11-04 um 13 39 07" src="https://user-images.githubusercontent.com/34510341/140314983-380b66aa-e76b-407a-a536-fa39b95e8a16.png">

**Expected behavior**
Should be named MEV slicer and maybe rephrase to Play MEV Slicer

**Comment by elena-zh** [comment]
Will be good to fix along with the #1764 issue in one PR

**Comment by elena-zh** [comment]
still relevant

**Comment by elena-zh** [comment]
I'm closing the issue as the button was renamed to the Cow Runner game, so the issue is no longer relevant.
